libs/log4cplus/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2017 OpenWrt.org
```

Comment:
Don't assign copyright to OpenWrt
See https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md